code/modules/jobs/job_types/wasteland.dm

```diff
@@ -868,3 +867,172 @@ Raider
 		/obj/item/reagent_containers/pill/patch/healpoultice = 2,
 		/obj/item/book/granter/crafting_recipe/tribal/bone = 1
 	)
+
+/datum/job/wasteland/f13bwsettler
+	title = "Blackwater Settler"
+	flag = F13BWSETTLER
+	total_positions = 8
+	spawn_positions = -1
+	faction = FACTION_WASTELAND
+	description = "You are a settler living on the frontier. Brahmin rustlers, raider gangs, and simple starvation threaten you, despite this, you've managed to settle down on the small frontier town of Blackwater."
```

Comment:
Being that they're close to the Khans, affirm a bit on their status together for both factions.

Reply:
Done, though I presumed Khans were slated for removal soon.

Reply:
Don't know when that will fully happen yet, so just as a precaution for now

Reply:
He closed that PR so yeah.